src/selectors/per-thread/markers.js

```diff
@@ -85,27 +89,35 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
     samples => MarkerData.deriveJankMarkers(samples, 50)
   );
 
-  const getReferenceMarkerTable: Selector<Marker[]> = createSelector(
+  const getReferenceMarkerTable: Selector<IndexedMarker[]> = createSelector(
     _getDerivedMarkers,
     _getDerivedJankMarkers,
-    (derivedMarkers, derivedJankMarkers) =>
-      [...derivedMarkers, ...derivedJankMarkers].sort(
+    (derivedMarkers, derivedJankMarkers) => {
+      const allMarkers = [...derivedMarkers, ...derivedJankMarkers].sort(
         (a, b) => a.start - b.start
-      )
+      );
+
+      // Magic Flow Transmutation
+      const resultMarkers: IndexedMarker[] = unsafeTransmute(allMarkers);
```

Comment:
Tell me what you think of this `unsafeTransmute` function. Would you prefer a more explicit flow incantation?

Reply:
I would prefer just a `(allMarkers: any)` here, with a short comment about why. I feel like this makes the code less obvious.